.github/styles/Vocab/Docs/accept.txt

```diff
@@ -117,13 +122,15 @@ Snapshotter
 softmax
 Splunk
 subcommand
+(subnet|subnets|Subnets)
 subtree
 TensorFlow
 (thanos|Thanos)
 Toree
 (traefik|Traefik)
 uncordon(|ed)
 unmount(|s|ing|ed)
+unpauses
```

Comment:
why do we need this? not really even a word.

Reply:
lol - I guess 'pause' is a specific thing for cluster objects, but I'll reword this. It just got caught because I edited the same page that had this word in there.